.github/workflows/validate-definition.yml

```diff
@@ -6,10 +6,10 @@ jobs:
     name: Validation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-node@v1
```

Comment:
Are these changes needed? The validation seemed to be "stuck" because somebody from our team had to approve your CLA

Reply:
@gvassallo I think these changes are needed to fix a validation error related to the node version. The repo was recently updated to use node 16: https://github.com/newrelic/entity-definitions/pull/810 -- and I had previously gotten an error about needing to upgrade the github actions package version.

Reply:
Alright.